frontend/src/types.ts

```diff
@@ -202,6 +202,38 @@ export type OdhDocument = {
   };
 };
 
+export type OdhQuickStart = {
+  metadata: {
+    name: string;
+    annotations?: { [key: string]: string };
```

Comment:
I have created this types for `OdhQuickStart`. So this should be kept up to date with backend types? or is it okay to have it only in frontend?

Reply:
This should extend `K8sResourceCommon` and be put into `k8sTypes` because it is a kubernetes resource.

It's fine just in the frontend. Sharing types with backend isn't done today.

Reply:
Does `OdhApplication` and `OdhDocument` also be put into `k8Types`? or is it okay to be in `types.ts`

Reply:
We're honestly not doing a great job of keeping things separate anyways. But since these are types related to CRDs, my understanding is they should reside in `k8sTypes`.
It's not necessary to refactor existing types as part of this change.